crt_portal/static/sass/custom/landing.scss

```diff
@@ -11,7 +18,7 @@ $arrow-height: 24px;
   }
 
   h2 {
-    margin-top: 1.5rem;
+    margin-top: 0;
```

Comment:
This effectively targets all of the `h2`s on the page, which results in `48px` of padding instead of `72px` of padding on desktop:
<img width="846" alt="Screen Shot 2020-08-12 at 10 20 38 AM" src="https://user-images.githubusercontent.com/509309/90040683-7cc0e980-dc85-11ea-89ff-bd8ba4fe7876.png">

vs the mocks:
<img width="865" alt="Screen Shot 2020-08-12 at 10 06 14 AM" src="https://user-images.githubusercontent.com/509309/90039218-93fed780-dc83-11ea-9b54-48ef7920236a.png">

So let's keep this rule at 1.5rem and write a more tightly scoped rule. Somewhere in this file, there's a `crt-landing--reporting_column` class. You want to write a rule that sets `.crt-landing--reporting_column h2` to a top margin of 0, and then you'll only target the elements you mean to!